tests/src/test/scala/neotypes/AsyncSuite.scala

```diff
@@ -1,12 +1,10 @@
 package neotypes
 
-import org.scalatest.Suites
-
 import scala.concurrent.{ExecutionContext, Future}
 import scala.reflect.ClassTag
 
 /** Testkit used to write specs abstracted from any concrete Async type. */
-abstract class AsyncTestkit[F[_]](implicit ct: ClassTag[F[_]]) {
+abstract class AsyncTestkit[F[_]](implicit ct: ClassTag[F[Any]]) {
```

Comment:
`ClassTag[F[_]]` cares only about outer `F` and I think `Any` is more precise version of wildcard.